framework/console/controllers/MigrateController.php

```diff
@@ -281,6 +313,7 @@ protected function generateMigrationSourceCode($params)
             'table' => $table,
             'fields' => $this->fields,
             'foreignKeys' => $this->foreignKeys,
+            'tName' => [$this, 'generatedTableName'],
```

Comment:
Instead of passing around function you can execute it and pass result around. That would be much cleaner.